packages/edit-site/src/components/page-pages/index.js

```diff
@@ -130,9 +130,8 @@ export default function PagePages() {
 			{
 				header: 'Status',
 				id: 'status',
-				cell: ( props ) =>
-					postStatuses[ props.row.original.status ] ??
-					props.row.original.status,
+				accessorFn: ( page ) =>
```

Comment:
So, apparently, when a field has an `accessorFn` the column is automatically made sortable, unless set otherwise. Fix at https://github.com/WordPress/gutenberg/pull/55210